4-AdvancedGrants/2-call-api-api-ca/MiddletierAPI/config.json

```diff
@@ -0,0 +1,23 @@
+{
+    "credentials": {
+        "tenantID": "Enter_the_Tenant_Id_Here",
+        "clientID": "Enter_the_Application_Id_Here",
+        "clientSecret": "Enter_the_Client_Secret_Here"
```

Comment:
>"clientSecret": "Enter_the_Client_Secret_Here" [](start = 8, length = 46)

comma added after client secret breaks the code

Reply:
This is something I need to fix in the Codegenerator :/